source/databases-collections.txt

```diff
@@ -0,0 +1,275 @@
+   mongoc_collection_set_read_prefs (collection, read_prefs);
+
+The following example shows how to change the read concern of a
+collection by using the ``mongoc_collection_set_read_concern()`` function:
+
+.. code-block:: c
+
+   mongoc_read_concern_t *read_concern = mongoc_read_concern_new ();
+   monogc_read_concern_set_level (read_concern, MONGOC_READ_CONCERN_LEVEL_LOCAL);
+   mongoc_collection_set_read_concern (collection, read_concern);
+
+The following example shows how to change the write concern of a
+collection by using the ``mongoc_collection_set_write_concern()`` function:
+
+.. code-block:: c
+
+   mongoc_write_concern_t *write_concern = mongoc_write_concern_new ();
+   mongoc_write_concern_set_w(write_concern, MONGOC_WRITE_CONCERN_W_MAJORITY);
+   mongoc_collection_set_write_concern(collection, write_concern);
+
+.. tip::
+
+   To see the types of available read preferences, see the
+   `API documentation <{+api-libmongoc+}/mongoc_read_prefs_t.html#read-modes>`__. 
+
+To learn more about the read and write settings, see the following guides in the
+{+mdb-server+} manual:
+
+- :manual:`Read Preference </core/read-preference/>`
+- :manual:`Read Concern </reference/read-concern/>`
+- :manual:`Write Concern </reference/write-concern/>`
+
+Tag Sets
+~~~~~~~~
+
+In the {+mdb-server+}, you can apply key-value :manual:`tags
+</core/read-preference-tags/>` to replica-set
+members according to any criteria you choose. You can then use
+those tags to target one or more members for a read operation.
+
+By default, the {+driver-short+} ignores tags
+when choosing a member to read from. To instruct the {+driver-short+}
+to prefer certain tags, use the ``mongoc_read_prefs_set_tags()`` function to set the
+tags in an instance of ``mongoc_read_prefs_t``. 
+
+In the following code example, the tag set passed to the ``mongoc_read_prefs_t`` instance
+instructs the {+driver-short+} to prefer reads from the
+New York data center (``'dc': 'ny'``) and to fall back to the San Francisco data
+center (``'dc': 'sf'``):
+
+.. code-block:: c
+
+   mongoc_read_prefs_t *read_prefs = mongoc_read_prefs_new (MONGOC_READ_SECONDARY_PREFERRED);
+   bson_t *tags = BCON_NEW ("DC", BCON_UTF8("ny"), "DC", BCON_UTF8("sf"));
+
+   mongoc_read_prefs_set_tags (read_prefs, tags);
+
+Local Threshold
+~~~~~~~~~~~~~~~
+
+If multiple replica-set members match the read preference and tag sets you specify,
+the {+driver-short+} reads from the nearest replica-set members, chosen according to
+their ping time.
+
+By default, the driver uses only those members whose ping times are within 15 milliseconds
+of the nearest member for queries. To distribute reads between members with
+higher latencies, include the ``localThresholdMS`` parameter in your connection string URI.
+
+The following example connects to a MongoDB deployment running on ``localhost:27017``
+and specifies a local threshold of 35 milliseconds:
```

Comment:
I: the code sets the local threshold to 100 milliseconds instead of 35

Reply:
Good catch